nbs/60_medical.imaging.ipynb

```diff
@@ -80,6 +80,56 @@
     "_all_ = ['DcmDataset', 'DcmTag', 'DcmMultiValue', 'dcmread', 'get_dicom_files']"
```

Comment:
<p>This doesn't show anything useful to the reader IMO</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/2732/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>deleted this cell, my thoughts on this was to show how <code>n_bins</code> is calculated but as you stated the cell above does not really show anything useful .  It's better displayed when n_bins is 1 so we can see that in this case 3 distinct buckets are generated, the beginning, the end and the number specified by <code>n_bins</code>  so in this case 1</p>